app/src/components/Loaders/InitializingTextSlider.tsx

```diff
@@ -2,42 +2,68 @@ import { Text } from '@chakra-ui/react';
 import { motion } from 'framer-motion';
 import { useEffect, useState } from 'react';
 
-const PHRASES = [
+import { shuffleArray } from '../../main/util';
+
+const PHRASES_IN_ORDER = [
+  'Initializing tables...',
   'Labeling contacts...',
-  'Discovering the meaning to life...',
   'Coloring graphs...',
 ];
 
+const PHRASES_SCRAMBLED = [
+  'Discovering the meaning to life...',
+  'Sifting through 🍆...',
+  'Analzying reactions...',
```

Comment:
You have the spell check extension on my guy?

Reply:
New computer. I don't. We should have corrected before merging!